pkg/rancher-desktop/utils/protocols.ts

```diff
@@ -78,6 +86,37 @@ export function setupProtocolHandler() {
 
     callback(result);
   });
+}
 
-  protocolRegistered.resolve();
+/**
+ * Set up protocol handler for x-rd-extension://
+ *
+ * This handler is used for extensions; the format is:
+ * x-rd-extensions://<extension id>/...
+ * Where the extension id is the extension image id, hex encoded (to avoid
+ * issues with slashes).  Base64 was not available in Vue.
+ */
+function setupExtensionProtocolHandler() {
+  protocol.registerFileProtocol('x-rd-extension', (request, callback) => {
```

Comment:
Is there meaning to the `x` in `x-rd-extension` or is this to avoid collision?

Reply:
I _thought_ there was an RFC about putting internal things with `x-`, but I couldn't find it — instead I found [RFC 6648](https://www.rfc-editor.org/rfc/rfc6648) which instead we _shouldn't_ do that.  I think it's still useful to denote that it's internal? (Nobody else should ever see this, and if they do we might need to change it.)

Reply:
We discussed out-of-band and agreed that the existing name is appropriate